assets/cubyz/biomes/grassland.zig.zon

```diff
@@ -48,7 +48,7 @@
 		},
 		.{
 			.id = "cubyz:sbb",
-			.structure = "cubyz:tree/oak/grassland_generator",
+			.structure = "cubyz:tree/oak/generator",
```

Comment:
Why did you merge the grassland and forest generators? I think they were supposed to behave differently.

Reply:
Only difference was in ratios between the children, they were slightly.